test/Mechanical/rotational.jl

```diff
@@ -98,8 +109,104 @@ end
 
     @named model = ODESystem(connections, t, systems=[fixed, torque, inertia1, idealGear, inertia2, spring, inertia3, damper, sine])
     sys = structural_simplify(model)
-    @test_broken prob = ODAEProblem(sys, Pair[], (0, 1.0))
-    # sol = solve(prob, Rodas4())
+
+    @test_throws Any prob = ODAEProblem(sys, Pair[], (0, 1.0))
+
+    @test_broken prob = DAEProblem(sys, D.(states(sys)) .=> 0.0, Pair[], (0, 10.0))
+    @test_skip begin
+        sol = solve(prob, DFBDF())
+        @test sol.retcode == :Success
+    end
 
     # Plots.plot(sol; vars=[inertia2.w, inertia3.w])
 end
+
+@testset "Stick-Slip" begin
+    function VelocityProfile(;name)
+        @named sine = Blocks.Sine(amplitude=10, frequency=0.1)
+        @named dz = Blocks.DeadZone(u_max=2)
+        @named lim = Blocks.Limiter(y_max=6)
+        @named output = Blocks.RealOutput()
+        connections = [
+            connect(sine.output, dz.input)
+            connect(dz.output, lim.input)
+            connect(lim.output, output)
+        ]
+        ODESystem(connections, t, [], []; name=name, systems=[sine, dz, lim, output])
+    end
+
+    @named fixed = Fixed()
+    @named spring = Spring(c=6.5)
+    @named damper = Damper(d=0.01)
+    @named inertia = Inertia(J=0.0001)
+    @named friction = RotationalFriction(f=0.001, tau_c=20, w_brk=0.06035, tau_brk=25)
+    @named vel_profile = VelocityProfile()
+    @named source = Speed(use_support=false)
+    @named angle_sensor = AngleSensor()
+
+    connections = [
+        connect(vel_profile.output, source.w_ref)
+        connect(source.flange, friction.flange_a)
+        connect(friction.flange_b, inertia.flange_a)
+        connect(inertia.flange_b, spring.flange_a, damper.flange_a)
+        connect(spring.flange_b, damper.flange_b, fixed.flange)
+        connect(angle_sensor.flange, inertia.flange_a)
+    ]
+
+    @named model = ODESystem(connections, t, systems=[fixed, inertia, spring, damper, vel_profile, source, friction, angle_sensor])
+    sys = structural_simplify(model)
+    prob = DAEProblem(sys, D.(states(sys)) .=> 0.0, Pair[], (0, 10.0))
+
+    sol = solve(prob, IDA())
+    @test sol.retcode == :Success
+    @test_broken sol[angle_sensor.phi.u] == sol[inertia.flange_a.phi]
+
+    @test_broken sol = solve(prob, dassl())
```

Comment:
Because it fails but IDA and DFBDF do not.

Reply:
that doesn't seem worth testing here. That's just that DASSL.jl is a bad. We know that and it's kind of irrelevant to real usage of MTK (which wouldn't use DASSL.jl)

Reply:
Ok I will remove it then.